src/lib/utils/config.ts

```diff
@@ -0,0 +1,43 @@
+function getConfigString (prop: string): string {
+    const val = process.env[prop] || null;
+
+    return val ? `${val}` : val!;
+}
+
+function getConfigNumber (prop: string): number {
+    const val = process.env[prop] || null;
+
+    return val ? parseFloat(`${val}`) : null || 1;
+}
+
+function getConfigBoolean (prop: string): boolean {
+    const val = process.env[prop] || null;
+
+    return val ? val === 'true' : false;
+}
+
+function getConfig (prop: string, type: 'string'): string;
+
+function getConfig (prop: string, type: 'number'): number;
```

Comment:
moglbys to w sumie zrobic bez overloada funckji tylko na generyku? @Haxikowy wez sie wypowiedz czy pierdole glupoty